contracts/Loan.sol

```diff
@@ -97,7 +98,7 @@ contract Loan is ZkAssetMintable {
     bytes32 noteHash,
     bytes memory metadata) = newTotal.extractNote();
 
-    logOutputNotes(mintedNotes);
+    // logOutputNotes(mintedNotes);
```

Comment:
meant to comment out?

Reply:
Good catch, shouldn't be commented out. Have pushed to correct